src/main/java/Duke.java

```diff
@@ -2,93 +2,166 @@
 import java.util.Arrays;
 
 public class Duke {
+    public static final String LOGO = " ______   _       _   _\n"
+            + "|  __  | | | ___ | | | | \n"
+            + "| |  | | | |/   \\| | | | \n"
+            + "| |__| | |   / \\   | | |____\n"
+            + "|______| |__/   \\__| |______|\n";
+    public static final String INVALID_MESSAGE = "Theres something wrong with your command! Type again!";
+    public static final String GREETING_MESSAGE = "SQUAWK! See you next time! :)";
+    public static final String LISTING_MESSAGE = "This are all the things I've remembered for you:";
+    public static final int VALID_ONE_PART_COMMAND = 1;
+    public static final int VALID_TWO_PART_COMMAND = 2;
+    public static final int MAX_TASK_LENGTH = 100;
```

Comment:
Good usage of Constant. Some other messages can also be shifted into constants as well. I suggest you create a separate Utils Class for this.